project/AppDependencies.scala

```diff
@@ -2,12 +2,12 @@ import sbt._
 
 object AppDependencies {
 
-  val scalaTestPlusPlayVersion = "3.1.3"
-  val mockitoVersion = "3.5.9"
-  val wireMockVersion = "2.27.2"
-  val customsApiCommonVersion = "1.53.0"
-  val simpleReactiveMongoVersion = "7.30.0-play-27"
-  val reactiveMongoTestVersion = "4.21.0-play-27"
+  val scalaTestPlusPlayVersion = "4.0.3"
+  val mockitoVersion = "3.11.1"
+  val wireMockVersion = "2.28.1"
+  val customsApiCommonVersion = "1.55.0"
```

Comment:
This should be 1.56.0 which will bring latest bootstrap and it might bring a few warnings
Also reactive mongo is 5.0 but keep play-27 not 26